build_modules/lib/src/meta_module_builder.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:build/build.dart';
+import 'package:glob/glob.dart';
+
+import 'meta_module.dart';
+
+/// The extension for serialized meta module assets.
+const metaModuleExtension = '.meta_module';
+
+/// Creates `.meta_module` file for any Dart library.
```

Comment:
nit: describe this a bit more `This module contains information about the full computed module structure for the package.`

Reply:
Done.